tox.txt

```diff
@@ -0,0 +1 @@
+tox==3.1.2
```

Comment:
Are you planning to add this to the pyup.io configuration?

Reply:
Yes; thanks for asking/reminding!

Would you prefer I do that here or in a separate PR?  Happy to do either (guessing the former?)

Reply:
Done; I ended up fixing the pyup.io config here.